back/app/services/permissions/idea_permissions_service.rb

```diff
@@ -39,7 +40,11 @@ def denied_reason_for_action(action, reaction_mode: nil, delete_action: false)
         # We preserved the behaviour that was already there, but we're not
         # sure if this is the desired behaviour.
         current_phase = @timeline_service.current_phase_not_archived project
-        IDEA_DENIED_REASONS[:idea_not_in_current_phase] if current_phase && !idea_in_current_phase?(current_phase)
+        return IDEA_DENIED_REASONS[:idea_not_in_current_phase] if current_phase && !idea_in_current_phase?(current_phase)
+
+        if action == 'reacting_idea' && IdeaStatus::REACTING_NOT_ALLOWED_CODES.include?(idea&.idea_status&.code)
```

Comment:
Should an idea always have a status? ie is `idea_status&` Equally `idea&` is probably redundant as you'd never get this far in the service if the idea did not exist